ncgen/bytebuffer.c

```diff
@@ -250,7 +250,7 @@ bbTailpeek(Bytebuffer* bb, char* pelem)
 char*
 bbDup(const Bytebuffer* bb)
 {
-    char* result = (char*)emalloc(bb->length+1);
+    char* result = (char*)malloc(bb->length+1);
```

Comment:
I never cast the result of mallocs. See https://stackoverflow.com/questions/605845/do-i-cast-the-result-of-malloc for reasons.